File: C/1020 - Idade em Dias.c
#include<stdio.h>
#include<stdlib.h>
#include<math.h>

int main() {
    int n,sum1,sum2,sum3;

    scanf("%d",&n);

    sum1=n/365;
    sum2=(n%365)/30;
    sum3=(n%365)%30;

    printf("%d ano(s)\n%d mes(es)\n%d dia(s)\n",sum1,sum2,sum3);

    return 0;
}

File: PYTHON/1020 - Idade em Dias.py
n1 = int(input())

sum1 = n1 // 365
n1 = n1 - sum1 * 365

sum2 = n1 // 30
n1 = n1 - sum2 * 30

print(f"{sum1} ano(s)")
print(f"{sum2} mes(es)")
print(f"{n1} dia(s)")